catalogue/webapp/components/ItemRequestModal/ItemRequestModal.tsx

```diff
@@ -106,7 +117,7 @@ const RequestDialog: FC<RequestDialogProps> = ({
   currentHoldNumber,
 }) => {
   const { enablePickUpDate } = useToggles();
-  const [pickUpDate, setPickUpDate] = useState<Date | undefined>();
+  const [pickUpDate, setPickUpDate] = useState<Moment | null | undefined>(null);
```

Comment:
Why do we need `null` and `undefined`? If it's necessary can we have a comment about it?

Reply:
We did. Setting null stops the calendar from selecting a default value. But I don't think we want this behaviour, so I'm going to take that out.